hedgehog/src/Hedgehog.hs

```diff
@@ -37,9 +37,9 @@
 --
 -- You can then load the module in GHCi, and run it:
 --
--- >>> tests
--- ━━━ Test.Example ━━━
---   ✓ prop_reverse passed 100 tests.
+-- > λ tests
```

Comment:
Add it your `~/.ghc/ghci.conf` for extra points 🚀 
```
:set prompt "\ESC[34m\STX%s\n\ESC[0;33m\STXλ \ESC[m\STX"
```
<img width="50" alt="screen shot 2017-04-12 at 1 23 38 pm" src="https://cloud.githubusercontent.com/assets/134805/24939817/2ea083ae-1f83-11e7-88fe-3fabed3382f8.png">